lms/templates/registration/password_reset_complete.html

```diff
@@ -6,49 +6,24 @@
 {% endblock %}
 
 {% block bodyextra %}
-  <script type="text/javascript">
-    $(function() {
-
-      // adding js class for styling with accessibility in mind
-      $('body').addClass('js');
-
-      // new window/tab opening
-      $('a[rel="external"], a[class="new-vp"]')
-      .click( function() {
-      window.open( $(this).attr('href') );
-      return false;
-      });
-
-      // form field label styling on focus
-      $("form :input").focus(function() {
-        $("label[for='" + this.id + "']").parent().addClass("is-focused");
-      }).blur(function() {
-        $("label").parent().removeClass("is-focused");
-      });
-    });
-  </script>
+<script type="text/javascript" src="{{STATIC_URL}}js/student_account/password_reset.js"></script>
 {% endblock %}
 
 {% block bodyclass %}view-passwordreset{% endblock %}
 
 {% block body %}
-  <section class="introduction">
-    <header>
-      <h1 class="title">
-        <span class="title-super">
-          {% trans "Your Password Reset is Complete" %}
-        </span>
-      </h1>
-    </header>
-  </section>
-
-  <section class="passwordreset container">
-    {% block content %}
-    <section role="main" class="content">
-      {% blocktrans with link_start='<a href="/login">' link_end='</a>' %}
-      Your password has been set.  You may go ahead and {{ link_start }}log in{{ link_end }} now.
-      {% endblocktrans %}
+<div id="password-reset-complete-container" class="login-register">
+    <section id="password-reset-complete-anchor" class="form-type">
+        <div id="password-reset-complete" class="form-wrapper" aria-hidden="true">
+            <div class="status submission-success" aria-live="polite">
+                <h4 class="message-title">{% trans "Your password reset is complete" %}</h4>
+                <ul class="message-copy">
+                {% blocktrans with link_start='<a href="/login">' link_end='</a>' %}
+                    Your password has been set. You may go ahead and {{ link_start }}log in{{ link_end }} now.
```

Comment:
Your password has been reset. {{ link_start }}Sign in to your account.{{ link_end }}